introspection.go

```diff
@@ -31,9 +31,10 @@ func NewService(serviceURL string, opts ...ClientOpt) *Service {
 }
 
 // Update queries the service's schema, name and version and updates its status.
-func (s *Service) Update() (bool, error) {
-	req := NewRequest("query brambleServicePoll { service { name, version, schema} }").
-		WithOperationName("brambleServicePoll")
+func (s *Service) Update(ctx context.Context) (bool, error) {
+	req := NewRequest("query brambleServicePoll { service { name, version, schema} }")
+	req.WithOperationName("brambleServicePoll")
```

Comment:
This is working because the `With...` Functions are modifying the request which is a pointer but the intention is for them to return a copy of the request, so I think we should leave it as chained method calls.